runtime/src/bank/metrics.rs

```diff
@@ -125,7 +125,11 @@ pub(crate) fn report_new_bank_metrics(
         ("blockhash_queue_us", timings.blockhash_queue_time_us, i64),
         ("stakes_cache_us", timings.stakes_cache_time_us, i64),
         ("epoch_stakes_time_us", timings.epoch_stakes_time_us, i64),
-        ("builtin_programs_us", timings.builtin_programs_time_us, i64),
+        (
+            "builtin_program_ids_us",
```

Comment:
This seems like unnecessary churn for anyone who might be trying to use these metrics before and after the change. I recommend leaving this unchanged.

Reply:
Alrighty I flipped this back to the original string.